formulaire/src/main/java/fr/openent/formulaire/export/FormQuestionsExportPDF.java

```diff
@@ -130,12 +131,36 @@ public void launch() {
                                         } else {
                                             // If already exist, add choice to JsonArray
                                             JsonArray choicesArray = question.getJsonArray(CHOICES);
-                                            // TODO A REPORTER ICI
                                             choicesArray.add(choice);
                                         }
                                     }
                                 }
 
+                                for (JsonObject q : mapQuestions.values()) {
```

Comment:
Optimisable sur les for et les if, avec des streams notamment etc. Dans le prolongement des opti de la 280 ça va être la même logique (models, stream, ...) ça simplifiera pas mal de choses 👍